File: source/includes/fts/search-index-management/drop-index.js
// connect to your Atlas deployment
const uri = "<connection-string>";

const client = new MongoClient(uri);

async function run() {
  try {
    const database = client.db("<databaseName>");
    const collection = database.collection("<collectionName>");

    // run the helper method
    await collection.dropSearchIndex("<index-name>");
    
  } finally {
    await client.close();
  }
}
run().catch(console.dir);

File: source/includes/fts/search-index-management/csharp/DropIndex.cs
using MongoDB.Bson;
using MongoDB.Driver;

// connect to your Atlas deployment
var uri = "<connection-string>";

var client = new MongoClient(uri);

var db = client.GetDatabase("<databaseName>");
var collection = db.GetCollection<BsonDocument>("<collectionName>");

// drop your Atlas Search index
collection.SearchIndexes.DropOne("<index name>");
